contracts/Test/Mocks/MockToken.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts (last updated v4.6.0) (token/ERC20/ERC20.sol)
+
+pragma solidity ^0.8.0;
+
+import { ERC20 } from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+contract MockToken is ERC20 {
+    uint8 private immutable _decimals;
```

Comment:
This line has some lint warnings. We should follow the lint rules or change the rule so it doesn't warn.

Reply:
We'll address this in VEN-2091